config/services.xml

```diff
@@ -4,26 +4,35 @@
            xsi:schemaLocation="http://symfony.com/schema/dic/services https://symfony.com/schema/dic/services/services-1.0.xsd">
 
     <services>
-        <defaults autowire="true" autoconfigure="true" />
+        <service id="meilisearch.engine" class="Meilisearch\Bundle\Engine">
+            <argument type="service" id="meilisearch.client" />
+        </service>
 
-        <prototype namespace="Meilisearch\Bundle\Command\" resource="../src/Command" />
+        <!-- After bumping to Symfony >5.1 deeprecate public services into private services -->
+        <service id="meilisearch.service" class="Meilisearch\Bundle\Services\MeilisearchService" public="true">
+            <argument /><!-- After bumping to Symfony 5.1 use type="abstract" -->
+            <argument type="service" id="meilisearch.engine" />
+            <argument type="collection" /><!-- After bumping to Symfony 5.1 use type="abstract" -->
+            <argument type="service" id="property_accessor" />
+        </service>
+        <service id="search.service" alias="meilisearch.service" public="true">
+            <deprecated package="meilisearch/search-bundle" version="0.14">The "%alias_id%" service alias is deprecated. Use "meilisearch.service" instead.</deprecated>
+        </service>
 
-        <service id="meilisearch.search_indexer_subscriber"
-                 class="Meilisearch\Bundle\EventListener\DoctrineEventSubscriber"
-                 public="true">
+        <service id="meilisearch.search_indexer_subscriber" class="Meilisearch\Bundle\EventListener\DoctrineEventSubscriber" public="true">
             <argument type="service" id="meilisearch.service" />
         </service>
         <service id="search.search_indexer_subscriber" alias="meilisearch.search_indexer_subscriber">
             <deprecated package="meilisearch/search-bundle" version="0.14">The "%alias_id%" service alias is deprecated. Use "meilisearch.search_indexer_subscriber" instead.</deprecated>
         </service>
 
         <service id="meilisearch.client" class="Meilisearch\Client" public="true" lazy="true">
-            <argument key="$url">%meili_url%</argument>
-            <argument key="$apiKey">%meili_api_key%</argument>
-            <argument key="$httpClient" type="service" id="psr18.http_client" on-invalid="ignore" />
-            <argument key="$clientAgents" type="collection">
-                <argument>%meili_symfony_version%</argument>
-            </argument>
+            <argument /><!-- url -->
```

Comment:
I'm curious, now we don't need to add the `key=""`?

Reply:
In the `src/DependencyInjection/MeilisearchExtension.php`, I saw that you are changing to use a positional instead of a named argument. How is this better?

Reply:
After PR, I left the comment about it :)